### PR TITLE
Updated hook section on Readme from PR313

### PR DESCRIPTION
**Pull request by Pabl0cks**
Reviewed descriptions and samples from Hook section on Readme from PR313 and tweaked them a bit to enhance clarity and consistency with the other sections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,36 +198,40 @@ const {
...
   eventName: "GreetingChange",
-  fromBlock: //the block number to start reading events from,
+  // Specify the starting block number from which to read events.
+  fromBlock: 31231,
   blockData: true,
-  filters: //filters to be applied to the event (parameterName: value),
-  transactionData: //if set to true it will return the transaction data for each event (default: false),
-  receiptData: //if set to true it will return the receipt data for each event (default: false),
+  // Apply filters to the event based on parameter names and values (parameterName: value),
+  filters: parameterName: value,
```
Comment (sverps) [inline README.md:205]: Maybe change this to something that is syntactically correct, e.g. 
```ts
// Apply filters to the event based on parameter names and values { [parameterName]: value },
filters: { premium: true }
```

**Review by sverps — CHANGES_REQUESTED** [review]
Nice improvement overall!

File: README.md (hunk 1 above)
Comment (Pabl0cks) [reply README.md]: Thx a lot for the review @sverps! Just fixed it

**Comment by carletex** [comment]
Looking good @Pabl0cks ! Just fixed a silly conflict with `main`. Thanks also @KcPele for your work on this!

**Review by carletex — APPROVED** [review]

**Pull request merged**